projekt_do_pracy_na_labach/fiszki_backend/deployment.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: backend-deployment
+  namespace: apka
+  labels:
+    app: backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: backend
+  template:
+    metadata:
+      labels:
+        app: backend
+    spec:
+      containers:
+      - name: backend-service
+        image: fiszli_backend:latest
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 80
```

Comment:
Tutaj powinno być raczej 8000 a tak to reszta git. Zapraszam po robienie zadanek na wyższą ocenę